services/hsts/hsts.service.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const label = 'hsts'
+const { BaseJsonService } = require('..')
+
+const Joi = require('joi')
+const schema = Joi.object({
+  status: Joi.string().required(),
+}).required()
+
+module.exports = class HSTS extends BaseJsonService {
+  static get category() {
+    return 'monitoring'
```

Comment:
Would `analysis` be a better category fit here? I've still not had a chance to dive into the details of this yet so honestly not sure what the best category would be